NitroxClient/MonoBehaviours/Gui/Settings/SettingsManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using NitroxClient.Unity.Helper;
+using NitroxModel.Logger;
+using Steamworks;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace NitroxClient.MonoBehaviours.Gui.Settings {
+    public class SettingsManager {
+        private static string Name;
+        private static string Key_Chat;
+        private static float Color_R;
+        private static float Color_G;
+        private static float Color_B;
```

Comment:
Do we need to store these channels separately?  If this color class is mutable we can just call setR()

Reply:
The `Color` class is mutable. It would be cool to make a wrapper property for a `Color` property (which stores it in a private field, and in the image if it exists), but because `Color` is a value-type (`struct`, *not* passed by reference), it's impossible to modify color channels separately. (eg `Color.r = 123;` would fail with a `Cannot modify the return value of 'SettingsManager.Color' because it is not a variable`). In this case, properties delegating the get/set to the fields of the private `Color` field might be desirable.